### DISCUSSION TITLE
Upcoming Meeting Widget really Meetings Later Today?

### DISCUSSION THREAD

**Discussion by willinrecovery** [code4recovery/12-step-meeting-list#1359] [Q&A] [answered]
Just tried the Upcoming Meetings widget and it's coming up empty.

Now it's Saturday evening and there are no more meetings entered for tonight but there is one entered for the morning.

Is Upcoming Meetings just "Meetings Later Today?"  Or have I got it connected wrong?

Thanks so much!

**Comment by anchovie91471** [comment, marked as answer]
Yes. I believe it's limited to the current day.

**Reply by joshreisner** [reply]
yes! it used to be the next five long ago but then it caused slow performance on some sites
